web/scripts/observations-backfill-calculated-cost.ts

```diff
@@ -0,0 +1,201 @@
+import "dotenv/config";
+
+import { prisma, Prisma } from "@langfuse/shared/src/db";
+
+type BackfillCalculatedGenerationCostParams = {
+  batchSize: number;
+  maxRowsToProcess: number | null;
+  sleepBetweenMs: number;
+};
+
+const backfillCalculatedGenerationCost = async (
+  params: BackfillCalculatedGenerationCostParams,
+) => {
+  try {
+    const { batchSize, maxRowsToProcess, sleepBetweenMs } = params;
+    log("Starting backfillCalculatedGenerationCost with params", params);
+
+    // Set the statement timeout to six hours
+    log("Setting statement_timeout to six hours...");
+    await prisma.$executeRaw(Prisma.sql`SET statement_timeout = '6h';`);
```

Comment:
do we need a statement length of 6h if the script below executes multiple db queries and not one only?